Description: Do not install upstart job file

Index: kpatch-0.9.7/contrib/Makefile
===================================================================
--- a/contrib/Makefile
+++ b/contrib/Makefile
@@ -6,9 +6,6 @@ install: all
 	$(INSTALL) -d $(SYSTEMDDIR)
 	$(INSTALL) -m 0644 kpatch.service $(SYSTEMDDIR)
 	sed -i 's~PREFIX~$(PREFIX)~' $(SYSTEMDDIR)/kpatch.service
-	$(INSTALL) -d $(UPSTARTDIR)
-	$(INSTALL) -m 0644 kpatch.conf $(UPSTARTDIR)
-	sed -i 's~PREFIX~$(PREFIX)~' $(UPSTARTDIR)/kpatch.conf
 
 uninstall:
 	$(RM) $(SYSTEMDDIR)/kpatch.service
