--- a/kpatch/kpatch	2023-02-20 15:31:32.597605441 +0100
+++ b/kpatch/kpatch	2023-02-20 15:42:22.594268104 +0100
@@ -355,7 +355,6 @@
 			i=$((i+1))
 			if [[ $i -eq $MAX_LOAD_ATTEMPTS ]]; then
 				die "failed to load module $module"
-				break
 			else
 				warn "retrying..."
 				sleep $RETRY_INTERVAL
--- a/kpatch-build/kpatch-cc	2023-02-20 15:51:58.017597894 +0100
+++ b/kpatch-build/kpatch-cc	2023-02-20 15:52:18.910931100 +0100
@@ -23,7 +23,7 @@
 			[[ "$obj" = */.tmp_mc_*.o ]] && break;
 
 			[[ "$obj" = */.tmp_*.o ]] && obj="${obj/.tmp_/}"
-			relobj=${obj##$KPATCH_GCC_SRCDIR/}
+			relobj=${obj##"$KPATCH_GCC_SRCDIR/"}
 			case "$relobj" in
 				*.mod.o|\
 				*built-in.o|\
--- a/kpatch-build/kpatch-build	2023-02-20 17:09:59.590902393 +0100
+++ b/kpatch-build/kpatch-build	2023-02-20 17:21:33.234231456 +0100
@@ -449,7 +449,7 @@
 	dir="$(dirname "$1")"
 	absdir="$(readlink -f "$dir")"
 	pwddir="$(readlink -f .)"
-	pdir="${absdir#$pwddir/}"
+	pdir="${absdir#"$pwddir/"}"
 	file="$(basename "$1")"
 	grepname="${1%.o}"
 	grepname="$grepname\\.o"
@@ -789,7 +789,7 @@
 		echo "Unpacking kernel source"
 
 		rpm -D "_topdir $RPMTOPDIR" -ivh "$SRCRPM" 2>&1 | logger || die
-		rpmbuild -D "_topdir $RPMTOPDIR" -bp --nodeps "--target=$(uname -m)" "$RPMTOPDIR"/SPECS/kernel$ALT.spec 2>&1 | logger ||
+		rpmbuild -D "_topdir $RPMTOPDIR" -bp --nodeps "--target=$(uname -m)" "$RPMTOPDIR/SPECS/kernel$ALT.spec" 2>&1 | logger ||
 			die "rpmbuild -bp failed.  you may need to run 'yum-builddep kernel' first."
 
 		if [[ "$DISTRO" = openEuler ]]; then
@@ -1161,6 +1161,7 @@
 
 		# create-diff-object orig.o patched.o parent-name parent-symtab
 		#		     Module.symvers patch-mod-name output.o
+		# shellcheck disable=SC2086
 		"$TOOLSDIR"/create-diff-object $CDO_FLAGS "orig/$i" "patched/$i" "$KOBJFILE_NAME" \
 			"$SYMTAB" "$SYMVERS_FILE" "${MODNAME//-/_}" \
 			"output/$i" 2>&1 | logger 1
@@ -1223,6 +1224,7 @@
 cd "$TEMPDIR/patch" || die
 
 # We no longer need kpatch-cc
+# shellcheck disable=SC2004
 for ((idx=0; idx<${#MAKEVARS[@]}; idx++)); do
     MAKEVARS[$idx]=${MAKEVARS[$idx]/${KPATCH_CC_PREFIX}/}
 done
@@ -1240,6 +1242,7 @@
 		extra_flags="--no-klp-arch-sections"
 	fi
 	cp -f "$TEMPDIR/patch/$MODNAME.ko" "$TEMPDIR/patch/tmp.ko" || die
+	# shellcheck disable=SC2086
 	"$TOOLSDIR"/create-klp-module $extra_flags "$TEMPDIR/patch/tmp.ko" "$TEMPDIR/patch/$MODNAME.ko" 2>&1 | logger 1
 	check_pipe_status create-klp-module
 	[[ "$rc" -ne 0 ]] && die "create-klp-module: exited with return code: $rc"
